.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,6 @@
+If possible, please include the following information when submitting an issue:
+
+1. All Openstack environment variables
+2. The output of using `gem list`
+3. The log files in your openstack_taster/logs directory
```

Comment:
So openstack_taster will actually just create a `logs` directory. The `openstack_taster` parent dir is not required for running openstack_taster once it is installed as a gem.